appveyor.yml

```diff
@@ -225,14 +228,20 @@ test_script:
      $systemTestOutput = (Resolve-Path "$testOutput\system")
      .\runsystemtests.bat --variable whichNVDA:installed --variable installDir:"${env:nvdaLauncherFile}" --include installer
      if($LastExitCode -ne 0) {
-      $errorCode=$LastExitCode
-      Add-AppveyorMessage "System test failure"
+      Set-AppveyorBuildVariable "testFailExitCode" $LastExitCode
+      Add-AppveyorMessage "FAIL: System tests. See test results for more information."
+     } else {
+      Add-AppveyorMessage "PASS: System tests."
      }
      Compress-Archive -Path "$systemTestOutput\*" -DestinationPath "$testOutput\systemTestResult.zip"
      Push-AppveyorArtifact "$testOutput\systemTestResult.zip"
      $wc = New-Object 'System.Net.WebClient'
      $wc.UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path "$systemTestOutput\systemTests.xml"))
-     if($errorCode -ne 0) { $host.SetShouldExit($errorCode) }
+
+after_test:
+ # fail build if any tests failed
+ - ps: |
+     if($env:testFailExitCode -ne 0) { $host.SetShouldExit($env:testFailExitCode) }

```

Comment:
Please compare a failing test pre/post this change. Look at how appveyor reports the failure to confirm it is the same.

Reply:
I did this here:

* https://ci.appveyor.com/project/NVAccess/nvda/builds/39304447/messages
* https://ci.appveyor.com/project/NVAccess/nvda/builds/39304636/messages

Reply:
Please include this information under the "testing strategy" heading, it is helpful as a reviewer. You did mention see this PR, but that requires the reviewer to investigate. Being clear just saves the reviewer time, and probably results in better testing. Outlining the cases that you have thought of, allows the reviewer to consider if there are other cases that might have been missed.

Reply:
updated the PR description to link explicitly to relevant build messages.